src/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+}
+
+.block {
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  flex-shrink: 0;
+  text-align: center;
+  line-height: 300px;
+}
+
+.block--1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+}
+
+.block--2 {
+  background-color: rgb(204, 0, 0);
+  order: 2;
+}
+
+.block--3 {
+  background-color: rgb(153, 0, 0);
+  order: 3;
+}
+
+.block--4 {
+  background-color: rgb(102, 0, 0);
+  order: 4;
+}
+
+.block--5 {
+  background-color: rgb(51, 0, 0);
+  order: 5;
+}
+
+.block--6 {
+  background-color: rgb(0, 0, 0);
+  order: 6;
+}
+
+@media only screen and (min-width: 481px) {
```

Comment:
check your breakpoints (i guess 600 and 900px)
and you have these when reaching `one column layout`:
![Snake](https://user-images.githubusercontent.com/39308717/165037765-6d516ab6-c717-43dc-b355-12f0b8e3c710.png)